scripts/import-programs.ts

```diff
@@ -127,15 +130,11 @@ fs.createReadStream(file)
                 }
             }
 
-            if (!data.id) {
-                console.error('No ID set for program, make sure to set an ID in the spreadsheet', data)
-                process.exit(1)
-            }
-
             states[code]!.programs.push({
-                id: data.id,
+                id: !data.id ? uuidv4() : data.id,
```

Comment:
I saw on the AC's was to generate a UUID. Do we expect the ID field in the CSV to be undefined? If we did would it be better to throw an error so we don't get data disparity between our JSON and the CSV?